ux.symfony.com/assets/styles/demos/live-memory.scss

```diff
@@ -0,0 +1,42 @@
+// -----------------------------------------------------------------
+// LiveMemory demo custom stylesheets
+//
+// Path: ux.symfony.com/assets/styles/demos/live-memory.scss
+//
+// Will be refactored with the rest of the website
+// -----------------------------------------------------------------
+
+@layer root, base, layout, component, utilities;
```

Comment:
To process this, we need a:

```
symfonycasts_sass:
    root_sass:
        - '%kernel.project_dir%/assets/styles/app.scss'
        - '%kernel.project_dir%/assets/styles/demos/live-memory.scss'
```

And then the stylesheet needs the `.scss` on the end:

```
<link rel="stylesheet" href="{{ asset('styles/demos/live-memory.scss') }}"/>
```

Reply:
What about building "locally" the CSS file ? Is this so crazy to imagine sass sources + final build CSS file.

And if anyone had to change something (and that 98% would be me i guess) just rebuild (phpstorm does it automatically) ? 

I'd like to _not_ use the Javascript file as _imho_ the browser is 100% doing well there... (agree to debate this another time/place ❤️ )

WDYT we see PageSpeed results and then decide ?

Reply:
> What about building "locally" the CSS file ? Is this so crazy to imagine sass sources + final build CSS file.

You have to convince me that Sass is needed at all :). Are you using some feature of it that won't work in CSS? Or are you really just using it to combine the many CSS files into one? And if so, do we care? What does PageSpeed say in that case? ;)

Reply:
Disabled sass for now (for this part). Probably won't need it anymore indeed. 

Now using the CSS file. 

I'd like to keep the sources in separated files to ease the debug next couple days, i _**promise you**_ i remove all that after.